lib/measure_repository_service_test_kit/library_data_requirements.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+require 'json'
+require_relative '../utils/data_requirements_utils'
+
+module MeasureRepositoryServiceTestKit
+  # tests for Library $package service
+  # rubocop:disable Metrics/ClassLength
+  class LibraryDataRequirements < Inferno::TestGroup
+    include GeneralUtils
+
+    title 'Library $data-requirements'
+    description 'Ensure measure repository service can execute the $data-requirements operation to the Library endpoint'
```

Comment:
Fixed in https://github.com/projecttacoma/measure-repository-service-test-kit/pull/10/commits/ec4d0221d72b55de2811077896dde9ec708e0f08 (only difference from your suggestion is that it says`Library` instead of `Measure`)